docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+
+***Note: Tell the reader WHY they would choose one choice over the other:***
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+> Note
+>
+> You can load data into StarRocks tables only as a user who has the INSERT privilege on those StarRocks tables. If you do not have the INSERT privilege, follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the INSERT privilege to the user that you use to connect to your StarRocks cluster.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+***Note: Processes that involve multiple components or steps may be easier to understand with a diagram. This example includes a diagram that helps describe the steps that happen when a user chooses the Broker Load option.***
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+> Note
+>
+> The examples use a sample dataset in Parquet format, if you want to load a CSV or ORC file, that information is linked at the bottom of this page.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`)
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+> Note
+>
+> The examples use IAM-based authentication. Other authentication methods are available and linked at the bottom of this page.
+
+Loading data from S3 requires having the:
+
+- S3 bucket
+- S3 object keys (object names) if accessing a specific object in the bucket
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/468f4dde-c4ad-4052-8c40-c32d05c7d4ae)
What say "S3 object keys (object names)"? It is a file path or URI? (In the syntax description, we refer to the file path or URI as file_path; see https://docs.starrocks.io/en-us/main/sql-reference/sql-statements/data-manipulation/BROKER%20LOAD#data_desc.) I think "S3 object key (object name)" is just the names of the object the user wants to access; but if the object is in a folder (or a subfoler of the folder) in the bucket, then just specifying the object key (object name) is not enough. For example, the object (named test_object) is located in a folder (named mydata) in the bucket (named bucket_s3), then the user must input s3a://bucket_s3/mydata/test_object in the load statement.

Reply:
Thanks @amber-create I agree that this is confusing. 

I dug in the S3 docs to find the official names and am using the AWS naming convention, so `object key (object name)` is correct for an object (file) including what AWS calls a `prefix` and we would call path, but yes--we can do better.

I added a line about this and a link the AWS specification.

One related note:
- Unless RD tells us differently, we should stop using `s3a` as Amazon recommends `s3` over `s3a`.  Do you know who we should check with?